package.json

```diff
@@ -18,6 +18,7 @@
     "bcrypt": "^5.0.0",
     "body-parser": "^1.19.0",
     "bootstrap": "^4.5.0",
+    "chart.js": "^2.9.3",
```

Comment:
Is this used anywhere?

Reply:
I think that's re-chart. Just goes by a different name in the package.json file.

Reply:
Removed "chart.js" from package.json